extensions/rocksdb-repos/tests/RepoTests.cpp

```diff
@@ -781,4 +780,83 @@ TEST_CASE("Content repositories are always running", "[TestRepoIsRunning]") {
   REQUIRE(content_repo->isRunning());
 }
 
+std::shared_ptr<minifi::FlowFileRecord> createFlowFileWithContent(core::ContentRepository& content_repo, std::string_view content) {
+  auto flow_file = std::make_shared<minifi::FlowFileRecord>();
+  const auto content_session = content_repo.createSession();
+  const auto claim = content_session->create();
+  const auto stream = content_session->write(claim);
+
+  stream->write(gsl::make_span(content).as_span<const std::byte>());
```

Comment:
we still use `gsl::span` in a few places, but I think `std::span` is preferred in new code:
```suggestion
  stream->write(utils::as_span<const std::byte>(std::span(content));
```